src/core/stdc/math.d

```diff
@@ -215,65 +215,152 @@ version( CRuntime_DigitalMars )
     }
   }
 }
-else version( CRuntime_Microsoft )
+else version( CRuntime_Microsoft ) // fully supported since MSVCRT 12 (VS 2013) only
 {
+  version( all ) // legacy stuff to be removed in the future
+  {
     enum
     {
-        ///
```

Comment:
Rebasing seems to have removed these comments. I guess these have been added so the symbols are shown in the online documenttion.